securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java

```diff
@@ -59,6 +62,16 @@ public static void setValue(@NonNull String key, int value,
         setValue(key, String.valueOf(value), context);
     }
 
+    @RequiresApi(Build.VERSION_CODES.JELLY_BEAN_MR2)
+    public static void setValue(@NonNull String key, Set<String> value,
+                                @NonNull Context context) throws CryptoException {
+        setValue(key + "_cnt", String.valueOf(value.size()), context);
```

Comment:
Cnt = count? if yes, could you call it count, please? And put it into a global variable as you use it later again.